### PR TITLE
chore(configs/eslint-config-suspensive): eslint correctly

### LINKED ISSUE #49: [BUG]: Vercel error
### **Package Scope**
all

Package name: all

### **Describe the bug**

image

### PR DESCRIPTION
**Pull request by manudeli**
fix #49 

# ESlint setup correctly
I've been putting off setting up eslint until now,
but it's time to clean up, so I set it up properly.

### New private packages
- eslint-config-suspensive

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

File: configs/eslint-config-supensive-common/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    'react/state-in-constructor': ['off'],
+    'react/prop-types': ['off'],
+    'react/require-default-props': ['off'],
+    'react/jsx-no-useless-fragment': ['off'],
+    'react/jsx-props-no-spreading': ['off'],
+    'react/no-unstable-nested-components': ['off'],
+    'react/function-component-definition': ['off'],
+    'react/no-unused-prop-types': ['off'],
+    'react/react-in-jsx-scope': ['off'],
+    'react/jsx-filename-extension': [1, { extensions: ['.js', '.jsx', '.ts', '.tsx'] }],
```
Comment (ChanhyukPark-Tech) [inline configs/eslint-config-supensive-common/.eslintrc.js:37]: There are a lot of things related to react in the eslint-config-suspensive-common
I wonder what the role of `eslint-config-suspensive-react` is and what is expected to be added in the future

**Review by ChanhyukPark-Tech — APPROVED** [review]
good 👍

File: configs/eslint-config-supensive-common/.eslintrc.js (hunk 1 above)
Comment (manudeli) [reply configs/eslint-config-supensive-common/.eslintrc.js]: I merged eslint-config-suspensive-common/react/next to `eslint-config-suspensive`

We can use like below
```js
// .eslintrc.js
module.exports = {
  root: true,
  extends: ['suspensive/react'], // we can use `suspensive/react` as subpath eslint-config-suspensive
  parserOptions: {
    tsconfigRootDir: __dirname,
    project: './tsconfig.json',
  },
}

```

I didn't know you could use `/` like `suspensive/react`, but I could. I got the idea when I saw that eslint-config-airbnb-typescript was using it like `airbnb-typescript/base`.

#### reference: [eslint-config-airbnb-typescript/base](https://www.npmjs.com/package/eslint-config-airbnb-typescript)
```js
// .eslintrc.js
module.exports = {
  root: true,
  extends: ['airbnb-base', 'airbnb-typescript/base']
}

```

File: .nvmrc [hunk 2]
```diff
@@ -1 +1 @@
-16.17
```
Comment (ChanhyukPark-Tech) [inline .nvmrc:1]: good👍

File: .vscode/extensions.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "recommendations": [
-    "arcanis.vscode-zipfs",
+    "streetsidesoftware.code-spell-checker",
```
Comment (ChanhyukPark-Tech) [inline .vscode/extensions.json:3]: goodgood👍

**Review by ChanhyukPark-Tech — APPROVED** [review]
thank you for rebuilding config setting!

**Pull request merged**